app/wama.web/Controllers/CheckInController.cs

```diff
@@ -60,6 +71,30 @@ public IActionResult WaiverNotSigned()
             return View();
         }
 
+        [HttpPost]
+        public async Task<IActionResult> Waiver(string signerName, string memberId)
+        {
+            var userAccount = await _UserAccountService.GetUserAccountAsync(memberId);
```

Comment:
We could do some more checks here. Can't necessarily think that UserAccount will always be there.

**_We don't have to deal with this now._**